cli/run_util.go

```diff
@@ -408,7 +408,7 @@ func executeStep(
 func runStep(
```

Comment:
NOTE: might the step input and environment processing could be a separate function:
- processEnviron(https://github.com/bitrise-io/bitrise/pull/721/files#diff-d630ad85efb836c79d7d9abc6172c355R442-R477)
- runStep would receive the evaluatedEnvironment only

This way you could call processEnviron() in the parent function (activateAndRunSteps) and you can do the stepInputExpansion for analytics outside runStep